Formula/uftrace.rb

```diff
@@ -0,0 +1,29 @@
+class Uftrace < Formula
+  desc "Function graph tracer for C/C++/Rust"
+  homepage "https://uftrace.github.io/slide/"
+  url "https://github.com/namhyung/uftrace/archive/refs/tags/v0.10.tar.gz"
```

Comment:
Most formulae use the shorted URL:
```suggestion
  url "https://github.com/namhyung/uftrace/archive/v0.10.tar.gz"
```

Reply:
Thanks, applied.